elevate-cpanel

```diff
@@ -557,17 +561,26 @@ sub do_check($self) {
     # cleanup the stage file
     unlink ELEVATE_STAGE_FILE;
 
-    if ($has_blockers) {
-        WARN( <<~'EOS' );
-        Please fix the detected issues before performing the elevation process.
-        EOS
+    if ($self->getopt('check')) {
+        if ($has_blockers) {
```

Comment:
Why adding a new `check-json` option and not using the current check to populate our generic json file with the blockers?
should not all we need is to add here something like?
```
update_stage_file( { 'blockers' => $self->{'blockers'} } );
```